src/server/scripts/Northrend/VioletHold/boss_xevozz.cpp

```diff
@@ -25,21 +25,23 @@ enum Spells
     SPELL_ARCANE_BARRAGE_VOLLEY                 = 54202,
     SPELL_ARCANE_BUFFET                         = 54226,
     SPELL_SUMMON_ETHEREAL_SPHERE_1              = 54102,
-    SPELL_SUMMON_ETHEREAL_SPHERE_2              = 54137,
+    SPELL_SUMMON_ETHEREAL_SPHERE_2              = 61337,
     SPELL_SUMMON_ETHEREAL_SPHERE_3              = 54138
 };
 
 enum NPCs
 {
     NPC_ETHEREAL_SPHERE                         = 29271,
-    //NPC_ETHEREAL_SPHERE2                      = 32582, // heroic only?
+    NPC_ETHEREAL_SPHERE2                        = 32582
 };
 
 enum CreatureSpells
 {
     SPELL_ARCANE_POWER                          = 54160,
+    H_SPELL_ARCANE_POWER                        = 59474,
     SPELL_SUMMON_PLAYERS                        = 54164,
-    SPELL_POWER_BALL_VISUAL                     = 54141
+    SPELL_POWER_BALL_VISUAL                     = 54141,
+    SPELL_POWER_BALL_DAMAGE_TRIGGER                = 54207
```

Comment:
Remove 3 spaces before ' = ' , to keep alignment with the line above (codestyle).